### ISSUE TITLE
Make Charlie's change: the 3rd input motor shouldn't turn on unless the claw is down and open

### ISSUE THREAD

**Issue by chlohal** [nhs-t10/Robotics_2021_2022#29]
so merry request that the third set of the flingy flingy spaghetti monsters [note: inputs] do not work unless the first player unless micro has just inputted the two sets of inputs or rather just and put that opens the claw maybe okay cuz unless claw is open the third thing you should not turn on I think that's good enough I mean the main point is unless the first call is also at the bottom position can you track that we can track that and yeah then calls at the bottom position and open okay then the third thing you should not work the question is I want to try that and there's a chance it's still bad but if it works as ideally then the block just gets it gets up in the robot but it gets stuck here so this person is like oh so even when you're holding it down it doesn't turn on it doesn't work so and then when it gets down it automatically the point is that the fact that the point is that I think it's good cuz I think the point doesn't work okay cool

**Comment by Mario-Runner** [comment]
TLDR: Don't spin the innermost noodle unless the claw is inside the robot and open.
